packages/flyout/src/Flyout.js

```diff
@@ -33,29 +33,40 @@ import PanelPresenter from "./presenters/PanelPresenter";
 
 const Flyout = props => {
   const [open, setOpen] = useState(props.defaultOpen);
+  const [actionRect, setActionRect] = useState(undefined);
```

Comment:
the unused `useState` hooks should be removed.

Reply:
@wmui51 It is not unused. It is being used in the useEffect() block.

Reply:
got it, sorry i missed that.